mock/world/builtinFunctionsMock.go

```diff
@@ -124,7 +123,7 @@ func addOutputTransferToVMOutput(
 	outTransfer := vmcommon.OutputTransfer{
 		Value:     big.NewInt(0),
 		GasLimit:  vmOutput.GasRemaining,
-		GasLocked: gasLocked,
+		GasLocked: 0,
```

Comment:
here you have 0 again.

Reply:
Leftover from debugging.
This function will be replaced by Camil shortly, anyway, but fixed it for good measure.